sdks/java/io/jdbc/src/main/java/org/apache/beam/sdk/io/jdbc/JdbcUtil.java

```diff
@@ -75,6 +77,23 @@ class JdbcUtil {
 
   private static final Logger LOG = LoggerFactory.getLogger(JdbcUtil.class);
 
+  static final Map<String, String> JDBC_DRIVER_MAP =
+      new HashMap<>(
+          ImmutableMap.of(
+              "mysql",
+              "com.mysql.cj.jdbc.Driver",
+              "postgres",
+              "org.postgresql.Driver",
+              "oracle",
+              "oracle.jdbc.driver.OracleDriver",
+              "mssql",
+              "com.microsoft.sqlserver.jdbc.SQLServerDriver"));
+
+  @VisibleForTesting
```

Comment:
Are we expecting users to call this outside of tests? If not, let's just test the built-in ones. If so, make it public with docs.

Reply:
A few of the tests read/write to a table (basically an E2E test) and use the derby embedded driver which I was not planning on exposing externally, but wanted a way to add it to the map for tests

Reply:
Oh, derby is an actual "fake/test" jdbc endpoint. I had totally missed this. Yes, this is fine.